apps/lead/filter_set.py

```diff
@@ -24,9 +24,13 @@ class LeadFilterSet(django_filters.FilterSet):
 
     ENTRIES_EXISTS = 'entries_exists'
     ASSESSMENT_EXISTS = 'assessment_exists'
+    ENTRIES_DO_NOT_EXIST = 'entries_do_not_exist'
+    ASSESSMENT_DO_NOT_EXIST = 'assessment_do_not_exist'
```

Comment:
Just being a grammar Nazi here :P

Reply:
Yes, makes sense.
Making this consistent with entries